python/test/t_CovarianceModel_std.py

```diff
@@ -81,67 +98,93 @@ def test_scalar_model(myModel):
 
 inputDimension = 2
 
-myDefautModel = ot.SquaredExponential([2.0], [3.0])
-print('myDefautModel = ',  myDefautModel)
-test_model(myDefautModel)
+# 1) SquaredExponential
+myModel = ot.SquaredExponential([2.0], [3.0])
+ott.assert_almost_equal(myModel.getScale(), [2], 1e-15, 1e-15)
+ott.assert_almost_equal(myModel.getAmplitude(), [3], 1e-15, 1e-15)
+test_model(myModel)
 
 myModel = ot.SquaredExponential([2.0] * inputDimension, [3.0])
+ott.assert_almost_equal(myModel.getScale(), [2, 2], 1e-15, 1e-15)
+ott.assert_almost_equal(myModel.getAmplitude(), [3], 1e-15, 1e-15)
 test_model(myModel)
 
 
-myDefautModel = ot.GeneralizedExponential([2.0], [3.0], 1.5)
-print('myDefautModel = ',  myDefautModel)
-test_model(myDefautModel)
+# 2) GeneralizedExponential
+myModel = ot.GeneralizedExponential([2.0], [3.0], 1.5)
+ott.assert_almost_equal(myModel.getScale(), [2], 1e-15, 1e-15)
+ott.assert_almost_equal(myModel.getAmplitude(), [3], 1e-15, 1e-15)
+ott.assert_almost_equal(myModel.getP(), 1.5, 1e-15, 1e-15)
+test_model(myModel)
 
 myModel = ot.GeneralizedExponential([2.0] * inputDimension, [3.0], 1.5)
+ott.assert_almost_equal(myModel.getScale(), [2, 2], 1e-15, 1e-15)
+ott.assert_almost_equal(myModel.getAmplitude(), [3], 1e-15, 1e-15)
+ott.assert_almost_equal(myModel.getP(), 1.5, 1e-15, 1e-15)
```

Comment:
```suggestion
ott.assert_almost_equal(myModel.getScale(), [2, 2], 0.0, 0.0)
ott.assert_almost_equal(myModel.getAmplitude(), [3], 0.0, 0.0)
ott.assert_almost_equal(myModel.getP(), 1.5, 0.0, 0.0)
```
I think you could do a search for "1e-15" and replace most of them with "0.0".

Reply:
It is useless. I understand that we might use `0` but it is just a way to keep small values